plugins/tvShowsApi.ts

```diff
@@ -0,0 +1,27 @@
+import { Context } from '@nuxt/types';
+import { TvShowsApi } from '~/api/api';
+import { Configuration } from '~/api/configuration';
+import { PluginInjection } from '~/types/utils';
+
+declare module '@nuxt/types' {
+  interface Context {
+    $tvShowsApi: TvShowsApi;
```

Comment:
should just use shows :p the prefix isn't required and it's more specific than it needs to be

Reply:
I don't disagree, but we should keep the same name as whatever the class in `api.js` is. It makes it easier to track it :yum: 

Down the line, once the API is rewritten, we'll have to rework some of the plugins anyway, so we can rename at that point, I think.

Reply:
I think that keeping it as same as the API will make it easier to keep track of. As it was mentioned, it can always be changed with the API rewrite